takeout: ignore placevisit events

purarue opened the pull request:
this might `ImportError` for people if they have not upgraded `google_takeout_parser`... should a warning be added there?

purarue committed:
takeout: ignore placevisit events

karlicoss commented on src/promnesia/sources/takeout.py:
```diff
@@ -13,7 +13,7 @@ def index() -> Results:
     try:
         from my.google.takeout.parser import events
-        from google_takeout_parser.models import Activity, YoutubeComment, LikedYoutubeVideo, ChromeHistory, PlayStoreAppInstall, Location
+        from google_takeout_parser.models import Activity, YoutubeComment, LikedYoutubeVideo, ChromeHistory, PlayStoreAppInstall, Location, PlaceVisit
```
Yeah you're right about ImportError. Maybe we should make it `_seen: Set[str]` instead and use class names there instead of actual types? Then the import won't even be necessary (along with PlayStoreAppInstall and Location)

purarue committed:
sources/takeout: use type name for seen set

this prevents import errors if the user
has not recently updated the
google_takeout_parser
